Scripts/Python/Cleft.py

```diff
@@ -148,25 +142,32 @@ def __init__(self):
 
     def OnFirstUpdate(self):
         pass
-        #~ # test for first time to play the intro movie
-        #~ vault = ptVault()
-        #~ entry = vault.findChronicleEntry(kIntroPlayedChronicle)
-        #~ if entry is not None:
-            #~ # already played intro sometime in the past... just let 'em play
-            #~ PtSendKIMessage(kEnableKIandBB,0)
-        #~ else:
-            #~ # make sure the KI and blackbar is still diabled
-            #~ PtSendKIMessage(kDisableKIandBB,0)
-            #~ # It's the first time... start the intro movie, just by loading the movie dialog
-            #~ PtLoadDialog("IntroMovieGUI")
-
+        # test for first time to play the intro movie
+        vault = ptVault()
+        entry = vault.findChronicleEntry(kIntroPlayedChronicle)
+        if entry is not None:
+            # already played intro sometime in the past... just let 'em play
+            PtSendKIMessage(kEnableKIandBB,0)
+        else:
+            # make sure the KI and blackbar is still diabled
+            PtSendKIMessage(kDisableKIandBB,0)
+            # It's the first time... start the intro movie, just by loading the movie dialog
+            PtLoadDialog("IntroMovieGUI")
 
     def OnServerInitComplete(self):
         global loadTomahna
         global fissureDrop
         
         ageSDL = PtGetAgeSDL()
-        
+        if StartInCleft():
+            ageSDL["clftYeeshaBookVis"] = (1,)
+            PtSendKIMessageInt(kUpgradeKILevel, kMicroKI)
+            PtSendKIMessage(kDisableEntireYeeshaBook,0)
+            PtFindSceneobject("microBlackBarBody", "GUI").draw.disable()
+            PtGetLocalAvatar().avatar.setDontPanicLink(True)
+        else:
+            ageSDL["clftYeeshaBookVis"] = (0,)
```

Comment:
One thing to note is that this will appear or disappear the Relto book in the Cleft for everyone based on the current player who is linking in. In the intended gameplay, this is completely fine, but it might be surprising to anyone using hacks to reach the Cleft with other players. Considering multiplayer Cleft not really an intended game mode, however, I'm inclined to not care.